components/elm/src/biogeochem/DryDepVelocity.F90

```diff
@@ -465,8 +465,20 @@ subroutine depvel_compute( bounds, &
                else
 
                   !Following Emmons et al (2020, JAMES), the stomatal resistances for shaded and sunlit
-                  !leaves should be added in parallel
-                  rs = 1.0_r8 / ( fsun(pi)/rssun(pi) + (1.0_r8 - fsun(pi))/rssha(pi) )
+                  !leaves should be added in parallel.  Also need to account for the conditions of
+                  !rssun=0.0 and/or rssha=0.0, for which only fsun=0 or fsun=1 is considered, respectively,
+                  !following the suggestion by Sam Silva.
```

Comment:
I would suggest removing `following the suggestion by Sam Silva.` from the code and adding it as a comment in the PR discussion.

Reply:
Good idea. Just did.